packages/react-dom/src/client/ReactDOMUseEvent.js

```diff
@@ -64,6 +65,9 @@ export function useEvent(
       priority = optionsPriority;
     }
   }
+  if (priority === undefined) {
+    priority = getEventPriorityForListenerSystem(type);
```

Comment:
Noticed this when refining the type here. We shouldn't do a lookup for priority if we supply one.